internal/backends/gl/glwindow.rs

```diff
@@ -672,6 +663,14 @@ impl PlatformWindow for GLWindow {
             )
         });
 
+        let mut result = Point::default();
+
+        let width = text_input.width() * scale_factor;
+        let height = text_input.height() * scale_factor;
+        if width <= 0. || height <= 0. {
+            return Rect::new(result, Size::new(1.0, font.height() / scale_factor));
```

Comment:
the 1.0 is not divided by the scale_factor here. just the font.height.
as written in the other comments, the font_request way doesn't work, since it returns None.

Reply:
Ah yes, sorry, I misread the parentheses :)